server/controllers/workSpace.controller.ts

```diff
@@ -0,0 +1,141 @@
+import { WorkspaceService } from "../services/workspace/workspace.service";
+import PromiseRouter from "express-promise-router";
+import { IRouterConfig } from "./router.interface";
+import { Request, Response } from "express";
+
+export class WorkspaceController {
+  constructor(
+    private readonly workspaceService: WorkspaceService,
+    private readonly baseUrl: string = "/sharkio/workspace",
+  ) {}
+
+  getRouter(): IRouterConfig {
```

Comment:
You haven't validated the values you get from the user. e.g empty name. 
Please do it when you develop the controllers.

Reply:
I have balidations in the client side, should i also add validations in the controller?

Reply:
Of course!
The validation in the FE are for UX. 
The validation in the BE are for avoiding errors and unpredictable behaviour.